src/lib/proposer/proposer.ml

```diff
@@ -375,14 +375,14 @@ let run ~logger ~prover ~verifier ~trust_system ~get_completed_work
                         [("state_hash", State_hash.to_yojson transition_hash)]
                       in
                       Logger.info logger ~module_:__MODULE__ ~location:__LOC__
-                        !"Submitting transition $state_hash to the transition \
-                          frontier controller"
+                        !"Submitting newly produced block $state_hash to the \
```

Comment:
`Successfully produced new block! Submitting block $state_hash to the transition frontier controller`